static/app/components/onboardingWizard/newSidebar.tsx

```diff
@@ -1,5 +1,5 @@
 import {Fragment, useCallback, useEffect, useMemo, useState} from 'react';
```

Comment:
in this file, I am introducing new design changes based on the [figma file ](https://www.figma.com/design/FOwcqOcD0Yrpwv0hBQGt3R/Specs%3A-Quick-Start-v2?node-id=187-3858&node-type=frame&t=SKR3RdfRTJVTLlRk-0) (sorry internal only)